scripts/superdesk-desks/desks.js

```diff
@@ -172,22 +172,22 @@
                 }
 
                 scope.$watch('filter', queryItems);
-                scope.$on('task:stage', function(event, data) {
+                scope.$on('task:stage', function(_e, data) {
                     if (scope.stage && (data.new_stage === scope.stage || data.old_stage === scope.stage)) {
                         scheduleQuery();
                     }
                 });
 
-                scope.$on('content:update', function($event, data) {
+                scope.$on('content:update', function(_e, data) {
                     if (cards.shouldUpdate(scope.stage, data)) {
                         scheduleQuery();
                     }
                 });
 
-                scope.$on('item:move', function($event, data) {
-                    if ((data.to_stage && data.to_stage === scope.stage) ||
-                        (data.from_stage && data.from_stage === scope.stage)) {
-                        scheduleQuery();
+                scope.$on('item:move', function(_e, data) {
+                    if ((data.to_desk && data.from_desk !== data.to_desk) ||
+                        (data.to_stage && data.from_stage !== data.to_stage))  {
+                        scheduleQuery(1000); // smaller delay.
```

Comment:
it should be a bit more, like 2s, it takes some time to elastic after a change to show the updates